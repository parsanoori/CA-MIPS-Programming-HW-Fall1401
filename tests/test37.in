8
362126225
178212521
536802894
935687152
287830741
516019616
891617136
592816860