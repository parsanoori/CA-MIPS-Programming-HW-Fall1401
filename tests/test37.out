178212521
287830741
362126225
516019616
536802894
592816860
891617136
935687152